src/test/java/org/gridsuite/study/server/NetworkModificationTreeTest.java

```diff
@@ -602,23 +608,43 @@ public void testGetLastParentNodeBuilt() throws Exception {
         assertEquals(node7.getId(), networkModificationTreeService.doGetLastParentNodeBuilt(node3.getId()));
     }
 
-    private void createNode(UUID studyUuid, AbstractNode parentNode, AbstractNode newNode) throws Exception {
+    private void createNode(UUID studyUuid, AbstractNode parentNode, NetworkModificationNode newNode) throws Exception {
         newNode.setId(null);
+
+        // Only for tests
```

Comment:
This seems a test to check if WRITE_ONLY mode works well ?
Do we keep it ?

Reply:
Yes
Comment has been completed